Problem compiling for Android

Not sure if this is the right place to ask this question: I'm trying to compile RxCpp for Android but I'm having difficulties as reported in this StackOverflow post:
https://stackoverflow.com/questions/34550833/compile-rxcpp-for-android
Do You have some idea of what's going on?
Thank You for Your support and Thank You for Reactive (I'm successfully using it on most of my C# project)!
Alessandro

I also confirm that adding `-DRXCPP_ON_IOS` also fix the `undefined reference to `__tls_get_addr'` problem.

Speaking of which, we can detect Android with `#ifdef __ANDROID__`. So, I think we need a permanent fix into the repo?

--- a/Rx/v2/src/rxcpp/rx-includes.hpp
+++ b/Rx/v2/src/rxcpp/rx-includes.hpp
@@ -93,6 +93,10 @@
 #endif
 #endif
 
+#if defined(__ANDROID__)
+#define RXCPP_ON_ANDROID
+#endif
+
 #pragma push_macro("min")
 #pragma push_macro("max")
 #undef min
@@ -127,7 +131,7 @@
 #include <typeinfo>
 #include <tuple>
 
-#if defined(RXCPP_ON_IOS)
+#if defined(RXCPP_ON_IOS) || defined(RXCPP_ON_ANDROID)
 #include <pthread.h>
 #endif
 

--- a/Rx/v2/src/rxcpp/rx-util.hpp
+++ b/Rx/v2/src/rxcpp/rx-util.hpp
@@ -7,7 +7,7 @@
 
 #include "rx-includes.hpp"
 
-#if !defined(RXCPP_ON_IOS) && !defined(RXCPP_THREAD_LOCAL)
+#if !defined(RXCPP_ON_IOS) && !defined(RXCPP_ON_ANDROID) && !defined(RXCPP_THREAD_LOCAL)
 #if defined(_MSC_VER)
 #define RXCPP_THREAD_LOCAL __declspec(thread)
 #else

